.github/workflows/integrate.yaml

```diff
@@ -109,6 +109,61 @@ jobs:
         timeout-minutes: 10
         run: "vendor/bin/phpunit"
 
+  code-coverage:
+    name: "Code Coverage"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+        dependencies:
+          - "highest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          # Needed for \ParaTest\Console\VersionProvider::getGitVersion
+          fetch-depth: 0
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "pcov"
+          php-version: "${{ matrix.php-version }}"
+          tools: composer:v1, cs2pr
+          extensions: sqlite
+
+      - name: "Get composer cache directory"
+        id: composercache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: "Cache dependencies"
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composercache.outputs.dir }}
+          key: ${{ runner.os }}-php-${{ matrix.php-version }}-${{ matrix.dependencies }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-php-${{ matrix.php-version }}-${{ matrix.dependencies }}-composer-
+
+      - name: "Install lowest dependencies"
+        if: ${{ matrix.dependencies == 'lowest' }}
+        run: "composer update --no-interaction --no-progress --prefer-lowest"
+
+      - name: "Install highest dependencies"
+        if: ${{ matrix.dependencies == 'highest' }}
+        run: "composer update --no-interaction --no-progress"
+
+      - name: "Run tests"
+        timeout-minutes: 10
+        run: "vendor/bin/phpunit --coverage-clover=coverage.xml"
+
+      - name: "Send code coverage report to Codecov.io"
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
@andreasschroth I'd like to add Code Coverage report for this library; an email to you to allow `codecov` application to this library should already been sent to you.

As far as I know, you should also go to https://codecov.io/gh/paratestphp/paratest, copy the token and add it to the secrets of Github Actions.

Can you check this out please?

Reply:
Wohaaa, everything seems working, so it seems I've enough permissions:

![immagine](https://user-images.githubusercontent.com/152236/88367711-b8eee180-cd8c-11ea-8ea3-97d853e39313.png)

Reply:
Mmm, nope, wwe still miss to give the authorization to Codecov Bot.

On PHPUnit repository, the report appears as such:

![immagine](https://user-images.githubusercontent.com/152236/88368100-9c06de00-cd8d-11ea-8e40-36f0d4efcf4a.png)

You can see the commenter is `bot` and the reports appear into Github Actions list.

Here instead the commenter is and outsider, and more important the reports do not appear in GA list:

![immagine](https://user-images.githubusercontent.com/152236/88368157-bf318d80-cd8d-11ea-91e9-816729f02551.png)

@andreasschroth may you have a look please?